src/cmds/net/telnetd/telnetd.c

```diff
@@ -169,16 +168,14 @@ static void *shell_hnd(void* args) {
 		MD(printf("dup2 STDERR_FILENO error: %d\n", errno));
 	}
 
-	ret = shell_run(shell_lookup("tish"));
-	if (ret != 0) {
-		MD(printf("shell_run error: %d\n", ret));
-	}
+	system("tish");
```

Comment:
Actually, I don't understand how CLOEXEC is blocking this? 
Isn't it could be emulated by manually closing files before exec?
Otherwise, good